certified-connectors/Revizto (Ireland)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1469 @@
+          "type": "string"
+        },
+        "firstname": {
+          "description": "The user's first name.",
+          "type": "string"
+        },
+        "id": {
+          "description": "The Revizto account ID.",
+          "type": "integer"
+        },
+        "lastname": {
+          "description": "The user's last name.",
+          "type": "string"
+        },
+        "role": {
+          "description": "This field is for internal use.",
+          "type": "string"
+        },
+        "uuid": {
+          "description": "The Revizto account UUID.",
+          "format": "uuid",
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "uuid",
+        "firstname",
+        "lastname",
+        "role",
+        "email"
+      ],
+      "title": "object",
+      "type": "object"
+    },
+    "ReportableSort": {
+      "properties": {
+        "direction": {
+          "description": "The sorting direction",
+          "enum": [
+            "asc",
+            "desc"
+          ],
+          "type": "string",
+          "x-nullable": true
+        },
+        "field": {
+          "enum": [
+            "assignee",
+            "clash_area",
+            "clashcount",
+            "clash_level",
+            "clash_room",
+            "clash_source_file",
+            "clash_space",
+            "clash_test",
+            "clash_zone",
+            "commented",
+            "created",
+            "customStatus",
+            "customType",
+            "deadline",
+            "id",
+            "priority",
+            "project",
+            "reporter",
+            "sheet",
+            "stamp",
+            "stamp_color",
+            "status",
+            "tags",
+            "type",
+            "unread",
+            "updated",
+            "public"
+          ],
+          "type": "string",
+          "x-nullable": true
+        }
+      },
+      "title": "ReportableSort",
+      "type": "object"
+    },
+    "RoleItem": {
+      "description": "The project member's project role.",
+      "properties": {
+        "id": {
+          "description": "The project role ID.",
+          "example": 12345,
+          "type": "integer"
+        },
+        "name": {
+          "description": "The project role name.",
+          "example": "Admin",
+          "type": "string"
+        },
+        "system": {
+          "example": 1,
+          "type": "integer"
+        },
+        "uuid": {
+          "description": "The project role UUID.",
+          "format": "uuid",
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "uuid",
+        "name",
+        "system"
+      ],
+      "title": "object",
+      "type": "object"
+    }
+  },
+  "host": "api.ireland.revizto.com",
+  "info": {
+    "description": "Facilitates seamless integration with the Revizto API (Ireland), empowering users for informed decision-making.",
+    "title": "Revizto API",
```

Comment:
Please remove API from the title, which is a reserved keyword. Please refer this link. https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info